lambify/docker-compose.yml

```diff
@@ -4,10 +4,15 @@ services:
     build:
       context: '.'
       dockerfile: Dockerfile-build
+    entrypoint: ./bin/entrypoint
     environment:
       - RAILS_ENV=${RAILS_ENV-development}
       - SAM_CLI_TELEMETRY=0
       - AWS_PROFILE=${AWS_PROFILE-default}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION-us-east-1}
+      - CI=${CI}
```

Comment:
FYI, in case you did not know, GitHub Actions now passes the `CI=true` now. It used to be we had to set this. This allows it to be passed down to all compose runners.